code/controllers/subsystem/jobs.dm

```diff
@@ -1,693 +1,695 @@
-				jobs_to_formats[job.title] = "disabled" // the job they already have is pre-selected
-			else if(!job.would_accept_job_transfer_from_player(M))
-				jobs_to_formats[job.title] = "linkDiscourage" // karma jobs they don't have available are discouraged
-			else if(job.total_positions && !job.current_positions && job.title != "Civilian")
-				jobs_to_formats[job.title] = "linkEncourage" // jobs with nobody doing them at all are encouraged
-			else if(job.total_positions >= 0 && job.current_positions >= job.total_positions)
-				jobs_to_formats[job.title] = "linkDiscourage" // jobs that are full (no free positions) are discouraged
-	return jobs_to_formats
-
-
-/datum/controller/subsystem/jobs/proc/log_job_transfer(transferee, oldvalue, newvalue, whodidit)
-	id_change_records["[id_change_counter]"] = list("transferee" = transferee, "oldvalue" = oldvalue, "newvalue" = newvalue, "whodidit" = whodidit, "timestamp" = station_time_timestamp())
-	id_change_counter++
-
-/datum/controller/subsystem/jobs/proc/slot_job_transfer(oldtitle, newtitle)
-	var/datum/job/oldjobdatum = SSjobs.GetJob(oldtitle)
-	var/datum/job/newjobdatum = SSjobs.GetJob(newtitle)
-	if(istype(oldjobdatum) && oldjobdatum.current_positions > 0 && istype(newjobdatum))
-		if(!(oldjobdatum.title in command_positions) && !(newjobdatum.title in command_positions))
-			oldjobdatum.current_positions--
-			newjobdatum.current_positions++
-
-
-/datum/controller/subsystem/jobs/proc/fetch_transfer_record_html(var/centcom)
-	var/record_html = "<TABLE border=\"1\">"
-
-	var/table_headers = list("Crewman", "Old Rank", "New Rank", "Authorized By", "Time")
-	var/hidden_fields = list("deletedby")
-	if(centcom)
-		table_headers += "<span class='bad'>Deleted By</span>"
-	record_html += "<TR>"
-	for(var/thisheader in table_headers)
-		record_html += "<TD><B>[thisheader]</B></TD>"
-	record_html += "</TR>"
-
-	var/visible_record_count = 0
-	for(var/thisid in id_change_records)
-		var/thisrecord = id_change_records[thisid]
-
-		if(thisrecord["deletedby"] && !centcom)
-			continue
-
-		record_html += "<TR>"
-		for(var/lkey in thisrecord)
-			if(lkey in hidden_fields)
-				if(centcom)
-					record_html += "<TD><span class='bad'>[thisrecord[lkey]]<span></TD>"
-				else
-					continue
-			else
-				record_html += "<TD>[thisrecord[lkey]]</TD>"
-		record_html += "</TR>"
-		visible_record_count++
-
-	record_html += "</TABLE>"
-
-	if(!visible_record_count)
-		return "No records on file yet."
-	return record_html
-
-/datum/controller/subsystem/jobs/proc/delete_log_records(sourceuser, delete_all)
-	. = 0
-	if(!sourceuser)
-		return
-	var/list/new_id_change_records = list()
-	for(var/thisid in id_change_records)
-		var/thisrecord = id_change_records[thisid]
-		if(!thisrecord["deletedby"])
-			if(delete_all || thisrecord["whodidit"] == sourceuser)
-				thisrecord["deletedby"] = sourceuser
-				.++
-		new_id_change_records["[id_change_counter]"] = thisrecord
-		id_change_counter++
+SUBSYSTEM_DEF(jobs)

```

Comment:
Why was this re-encoded?

Reply:
I have NO IDEA but now i got to figure out how to revert it out.

Reply:
the last one to even touch that one was https://github.com/ParadiseSS13/Paradise/commit/7083aec682994d0b9d2d124e002f3118dc4c4d24 so if i can figure out how to revert it out or where it got redid we would be good. I am having an issue finding it *i had trouble fixing this before i don't want to remake the whole pr*